spec/support/jasmine.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Isn't there a way to inline this in the gulpfile?
I don't like to have useless files around